doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -50,51 +50,52 @@ Synopsis
 ========
```

Comment:
Suggest adding a note/warning above Synopsis about the change in syntax for the mount command. Someone may be confused by existing scripts that mount using the old syntax.